app/controllers/spree/api/retailops/settlement_controller.rb

```diff
@@ -323,29 +335,34 @@ def pick_payment(&blk)
           # new state so that RetailOps can reconcile its own notion of the
           # payment state.
           def settle_payments_if_desired
+            partial_settle(capture: true, partial_capture: true, void: true, refund: true)
```

Comment:
Not sure this will work because partial_settle assumes stringified hash keys/params:

```ruby
irb(main):008:0> def test(ops)
irb(main):009:1>   p ops['capture']
irb(main):010:1> end
=> :test
irb(main):011:0> test(capture: true, partial_capture: true, void: true, refund: true)
nil
=> nil
```